Clinical-Genomics/cgbeacon2
# Parse and save SVs to beacon database

fix #6

How to test this PR:
- Having a database already containing a dataset `test_ds`
- Run the following command:
`cgbeacon2 add variants -ds test_ds -vcf cgbeacon2/resources/demo/643594.clinical.SV.vcf.gz -sample ADM1059A1`
- 9 structural variants should be saved to database.
- The command to delete all variants for a sample should remove them all.
